src/lib/api.js

```diff
@@ -20,7 +20,7 @@ const getErrorMessage = async ({ response }) => {
 
 const checkResponse = async ({ response }) => {
   if (!response.ok) {
-    const message = await getErrorMessage({ response }).catch(() => undefined)
+    const message = await getErrorMessage({ response }).catch(() => {})
```

Comment:
How does this work with https://eslint.org/docs/rules/no-empty-function?
I actually prefer the explicit `undefined` in this case

Reply:
This is compatible with `no-empty-function`.
Note that this rule catches more cases, such as `return undefined` and `arg = undefined`.

Would you prefer to leave to:
  - leave this rule enabled, and use `=> {}`
  - leave this rule enabled, and use `=> undefined` + `eslint-disable-next-line` (there is no rule option for that specific use case unfortunately) 
  - disable this rule

I am good with any of those solutions, just let me know :)

Reply:
Let's keep it then